app/assets/javascripts/analytics/static-analytics.js

```diff
@@ -188,7 +188,12 @@
   };
 
   StaticAnalytics.prototype.setUserJourneyStage = function(pageType) {
-    this.setDimension(33, pageType);
+    // Track the stage of a user's journey through GOV.UK. If the page does not
+    // set a page type in a meta tag, default to "thing", which identifes a page
+    // as containing content rather than some kind of navigation.
+    var defaultUserJourneyStageValue = "thing";
+    var dimensionValue = (typeof pageType === "undefined") ? defaultUserJourneyStageValue : pageType;
```

Comment:
Is there any reason in particular you're checking specifically for `undefined` rather than using `!pageType` ? Or do you specifically want to allow eg `pageType = null` or `pageType = ''` (and if so, what do these mean?)